docs/adrs/ADR-1.md

```diff
@@ -13,29 +12,97 @@ Deepansh
+On every run, the cron job will select the sessions with the following filters:
+- Sessions which are either infinite sessions or non infinite sessions which have ended
+- Sessions with `test_type` as `homework`
+
+The selected session ids will be sent to an SNS notification, which will trigger a Lambda function implementing the ETL flow. The ETL flow will extract the required data, transform it, and dump into the required tables.
+
 ## Context & Problem Description
 > Go into detail about the subject in question. Why is this decision important? What problems are there that this decision needs to be made? The urgency of the decision? Datapoints and related background information. Vocabulary and key terms.
 
+Till now we have only been syncing assessment sessions but not homework sessions. For homework sessions, the extracted data will come from two places: **Firestore Sessions DB** and **Quiz Mongo DB**.
+The data being extracted from the **Firestore Sessions DB** is the data pertaining to the sessions. Things like `program`, `grade`, `course`, `stream` and many other, basically the column values stored in the **Quiz Session Creator** google sheet. The data being extracted from the **Quiz Mongo DB** will be the responses of students who have attempted that quiz and the related metadata. Decisions regarding some questions mentioned below need to be taken:
+1. Where should the homework sessions data be stored on BQ? Should it be stored in the `test_responses` table that exists or some new table?
+2. If storing in a different table, how should we name the new table? The old one is `test_responses`. It doesn't have the keyword `assessments` in the name.
+3. Is the question level analysis also needed for homework tests? If yes, then should it be stored in a separate table or in the table for `assessments`?
+4. This data also needs to go into the Dynamo DB (for reporting engine). Should it go to a different table or the same existing one?
+5. What all fields are necessary or good to have for the homework level sessions? The fields listed below are the ones present in the assessments table. We can filter out or add more fields for the homework quizzes. Decision needs to be taken on entries with a :question: emoji:
+
+    | Field | Comments |
+    | ------ | ----------- |
+    | user_id | will be added |
+    | num_skipped | :question: Right now one can't skip in homework mode. But it's possible in the future, so for now, should we keep it and the value will be 0? |
+    | num_correct | :question: Number of correctly answered graded questions. This will not include ungraded questions right? |
+    | num_wrong | :question: Number of wrongly answered graded questions. This will not include ungraded questions right? |
+    | marks_scored | :question: Marks scored by the user on graded questions wrt marking scheme. What is the default marking scheme if nothing is present? |
```

Comment:
Is it possible for marking scheme to not be present in the current quiz creation API? If so, we should change that, even if it means putting a default in the *quiz engine*. The ETL flow should not assume anything here

Reply:
yes, marking_scheme has a value of `None` by default. So not required for the API. 
Makes sense, the default value should not be `None` but a default decided marking scheme